src/components/searchResultsAddInfoPanel/searchResultsAddInfoPanel.jsx

```diff
@@ -3,46 +3,86 @@ import { hot } from "react-hot-loader";
+import PropTypes from "prop-types";
+import { changeSortOption } from "../../actions/actions";
+import { connect } from "react-redux";
 
 
 class SearchResultsAddInfoPanel extends React.Component {
-    constructor (props) {
-        super(props);
-        this.state = {}; // is necessary to show search results info or film by sorting; change on sort switching
-    }
-
     render () {
+        const {
+            total,
+            changeCurrentSortOption,
+            selectedFilterOption,
+            selectedSortOption,
+            searchInputValue,
+            moviesList,
+            selectedMovieInfo
+        } = this.props;
+
+        const sortOptionsGroup = [
+            {
+                id: sortOptions.RELEASE,
+                label: "release date",
+                checked: selectedSortOption === sortOptions.RELEASE
+            },
+            {
+                id: sortOptions.RATING,
+                label: "rating",
+                checked: selectedSortOption === sortOptions.RATING
+            }
+        ];
+
         return (
             <div className="searchResultsAddInfoPanel">
                 <div className="container">
-                    {/*TODO: add if statement and unit test*/}
-                    <div className="searchResultsAddInfo">
-                        <p>7 movies found</p>
-                        <RadioGroup title="Sort by"
-                            name="sortByField"
-                            radios={[
-                                {
-                                    id: "releaseDate",
-                                    label: sortOptions.RELEASE,
-                                    defaultChecked: true
-                                },
-                                {
-                                    id: "rating",
-                                    label: sortOptions.RATING
-                                }
-                            ]}
-                        />
-                    </div>
-                    <div className="searchResultsAddInfo">
-                        <div className="searchResultsAddInfo__displayInfo">
-                            <span>Films by</span>
-                            <span className="searchResultsAddInfo__displayInfoValue">Drama genre</span>
+                    {(moviesList.length > 0 && Object.keys(selectedMovieInfo).length === 0) &&
+                        (
+                            <div className="searchResultsAddInfo">
+                                <p>{total} movies found</p>
+                                <RadioGroup title="Sort by"
+                                    name="sortByField"
+                                    handleAction={changeCurrentSortOption}
+                                    radios={sortOptionsGroup}
+                                />
+                            </div>
+                        )}
+                    {Object.keys(selectedMovieInfo).length > 0 &&
```

Comment:
`Object.keys(selectedMovieInfo).length > 0` can be moved to a separate const as you use it twice

Reply:
done